en/app.ftl

```diff
@@ -232,6 +232,25 @@ landing-use-cases-gaming = Gaming
 landing-use-cases-gaming-body-2 = Enjoy online gaming, but worried about other gamers or bullies learning who you are through your gamertag and email address?
     Use an email mask to put one more layer of protection between your online identity and the games you play online.
 
+landing-use-cases-shopping-hero-heading = Shopping with email masks
+landing-use-cases-shopping-hero-content1 = Want to buy something online and don’t know or fully trust the shop?
+landing-use-cases-shopping-hero-content2 = Use an email mask whenever you shop online. Get the confirmation sent to your real email and then easily turn the mask off anytime later.
+
+landing-use-cases-on-the-go = On the Go
+landing-use-cases-on-the-go-heading = On the go with { -brand-name-relay }
+landing-use-cases-on-the-go-lead = Instantly make a custom email mask anywhere and everywhere you go!
+landing-use-cases-on-the-go-connect-heading = Connect on the go
+landing-use-cases-on-the-go-connect-body = Use your email mask when you want to privately sign into your favorite coffee shop or public Wi-Fi
```

Comment:
Noticed this only when translating: this should have been "sign in to" https://mana.mozilla.org/wiki/pages/viewpage.action?spaceKey=FIREFOX&title=Word+List#WordList-signin